### ISSUE TITLE
how to preserve column widths after data refresh?

### ISSUE THREAD

**Issue by topchyan** [ragardner/tksheet#84]
In my application I am trying to refresh the sheet with new data (filtered) but after that operation column widths are not preserved and go back to the default (I think it's 120). What can be done to preserve the column width upon refresh?

**Comment by ragardner** [comment]
If you're using `set_sheet_data()` you can use arguments `reset_col_positions = False` and `reset_row_positions = False`

Let me know if this does not answer your question

**Comment by topchyan** [comment]
Here's what I attempted to do in my code...

```   
    (h, d) = get_data(sql)

    w = sheet.get_column_widths(canvas_positions = False)
    sheet.headers = h  
    sheet.set_column_widths(column_widths = w)
    sheet.set_sheet_data(d)
```
Not sure if the sequence of these commands is quite right, since it's not obvious what the recommended way to interact with a grid is once it is populated. For example I tried to create a button that would change column widths on filled grid, but that didn't do much.

**Comment by topchyan** [comment]
The issue seems to be addressed in 5.34. Thank you!
